onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -495,6 +495,18 @@ def _where_input_wrangler(
         reason="atleast_3d_single_tensor overload takes single tensor as input",
     ),
     TorchLibOpInfo("baddbmm", core_ops.aten_baddbmm),
+    TorchLibOpInfo(
+        # This string is a unique ID. In extra_opinfo.py, we
+        # also define test data for this ID with
+        # `opinfo_core.OpInfo("aten.bernoulli.p", ...)`.
```

Comment:
```suggestion
        # Defined in extra_opinfo.py
```

to simplify?

Reply:
It's too simplified. I only put minimal but precise words to help me to remember what should be done. `Defined in extra_opinfo.py` doesn't say much.

Reply:
Maybe move this info to the top of the file to improve the instructions?

Reply:
Probably after I fix all other DORT and distributed inference bugs... 
Those are notes that indicate missing information. If I remove it, it will be hard to retrieve what is missed when a naive dev wants to write a new test. If I over-simplify it, it will be hard to see the exact missing concept. The instructions need a big revision as I don't see code example with it. The scope of this PR is small --- support aten::bernoulli for llama. I'd defer other efforts to other PRs.

Reply:
SG